Incompatibility with `image` 0.25.0

Updating to `image` 0.25.0 causes an issue at the trait bound:

```rs
let mut image = rqrr::PreparedImage::prepare(image);
```

The following compilation error occurs:

```text
error[E0277]: the trait bound `image::ImageBuffer<image::Luma<u8>, std::vec::Vec<u8>>: image::image::GenericImage` is not satisfied
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -28,10 +28,10 @@ required-features = ["img"]
 harness = false
 
 [dev-dependencies]
-criterion = "0.4"
-image = { version = "0.24" }
+criterion = "0.5"
+image = { version = "0.25" }
 
 [dependencies]
 g2p = "1.0"
-lru = "0.9"
-image = { version = "0.24", optional = true, default-features = false }
+lru = "0.12"
+image = { version = "0.25", optional = true, default-features = false }

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,7 @@
 [package]
 name = "rqrr"
-edition = "2018"
+edition = "2021"
+rust-version = "1.67.1"
 version = "0.6.0"
 authors = ["WanzenBug <moritz@wanzenbug.xyz>"]
 license = "MIT OR Apache-2.0"

